app/main.py

```diff
@@ -16,4 +16,20 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+elements = [lucky_number, pi, one_is_a_prime_number, name,
+            my_favourite_films, profile_info, marks,
+            collection_of_coins]
+
+mutable = []
+immutable = []
+
+for elelment in elements:
+    if type(elelment) in (list, dict, set):
+        mutable.append(elelment)
+    else:
+        immutable.append(elelment)
```

Comment:
No additional logic is required - just pass the variables to the correct list in the dictionary.